feature/s3/manager/integ_upload_test.go

```diff
@@ -161,8 +161,9 @@ func TestInteg_UploadPresetChecksum(t *testing.T) {
 	}{
 		"auto single part": {
 			"no checksum": {
-				payload:    bytes.NewReader(singlePartBytes),
-				expectETag: singlePartETag,
+				payload:             bytes.NewReader(singlePartBytes),
+				expectChecksumCRC32: singlePartCRC32,
```

Comment:
1. The tests is called "no checksum" but it has an expected checksum. We should change the name of the test
1. Does it still makes sense to have a `no checksum` test

Reply:
I prefer to change its name to "no checksum algorithm passed"